app/lib/tax_return_status.rb

```diff
@@ -0,0 +1,31 @@
+class TaxReturnStatus
```

Comment:
@alaroia-cfa @sartan There are a couple of reasons that I resurrected the TaxReturnStatus class, albeit in a more simplified form.

First, I'm always trying to keep the single responsibility principle + Sandi Metz' "rules" in mind (https://thoughtbot.com/blog/sandi-metz-rules-for-developers) when writing Ruby classes, and one of the big ones is to keep classes less than 100 lines. The TaxReturn model was pushing 100 lines, and most of it had to do with methods dealing only with setting up statuses and stages. 

Second, because of the way I'm trying to calculate the value of a const outside of the context of a method using a method, the method needed to be defined BEFORE it was used in the class, and it was really strange to do that in TaxReturn class itself. 

Third, since we're using the stages in backend code to create scopes and facilitate filtering, it made sense to me to have the calculation of the stages outside of the helper which are more for presentational purposes. The determine_statuses_by_stage method is really similar to what was in the helper, but now it returns the raw stage name, which then gets translated in the helper.

Reply:
I'm also willing to move this back into TaxReturn -- I actually haven't moved the tests because I'm waiting for feedback before making a final decision.